pygaggle/rerank/transformer.py

```diff
@@ -247,3 +249,29 @@ def rescore(self, query: Query, texts: List[Text]) -> List[Text]:
             text.score = max(smax_val.item(), emax_val.item())
 
         return texts
+
+
+class CrossEncoderReranker(Reranker):
+    def __init__(self,
+                 pretrained_model_name_or_path='cross-encoder/ms-marco-MiniLM-L-2-v2',
+                 max_length=512,
+                 device=None,
+                 use_amp=None):
+        device = device or ('cuda' if torch.cuda.is_available() else 'cpu')
+        self.use_amp = use_amp or (device == 'cuda')
```

Comment:
Why would we want to default to amp if there is a gpu it could bring performance drop, more explicit a flag would be better.

Reply:
You're right, I'm changing it to disabled by default.